### ISSUE TITLE
Anti-plasminogen

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7238]
**Preferred term label:**
Anti-plasminogen binding peptide antibody positivity

**Synonyms**
Anti-PBP antibody positivity

**Definition (free text, please give PubMed ID)**
PMID: 30075546

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:103919

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
added
